boards/arm/phy62xx/phy6222/scripts/Make.defs

```diff
@@ -35,7 +35,7 @@ ifeq ($(CONFIG_DEBUG_SYMBOLS),y)
 endif
 
 ifneq ($(CONFIG_DEBUG_NOOPT),y)
-  ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
+  ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing -fno-strength-reduce -fomit-frame-pointer
```

Comment:
revert, don't need

Reply:
i will revert this part next submit

Reply:
why? please fix the problem first.

Reply:
this part will optimize the compile generated code.
do you sure let me remove?

Reply:
In the latest mainline, -fomit-frame-pointer is done in the common place:
https://github.com/apache/incubator-nuttx/blob/master/arch/arm/src/armv6-m/Toolchain.defs#L64-L68